### ISSUE TITLE
Minimum word count isn't validated server-side

### ISSUE THREAD

**Issue by tremby** [verbb/formie#1198]
### Describe the bug

Minimum word count isn't validated on submission.

### Steps to reproduce

1. Make a single-line text field
2. Set it to be limiting
3. Set minimum to 2 words
4. Save
5. Submit the form via the GQL mutation, with a single character in the single-line text field
6. See it is accepted

I expected to see server-side validation, as there is for character limits.

### Form settings

- Multi-page form: n/a
- Submission Method: GQL
- Client-side Validation: no but n/a
- Custom Form Templates: yes but n/a

### Craft CMS version

4.2.7

### Plugin version

2.0.18

### Multi-site?

yes but n/a

### Additional context

I didn't test maximum, but it's possible the issue exists there too.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.19](https://github.com/verbb/formie/releases/tag/2.0.19)
